project/Versions.scala

```diff
@@ -34,7 +34,7 @@ object Versions {
   val CommonsIO       = "2.4"
   val CommonsLang3    = "3.3.2"
   val Config          = "1.2.1"
-  val Guava           = "16.0.1"
```

Comment:
This seems to be delicate matter. Do you have any special corner cases on your mind that I should tests with new guava version?